config/kbin_routes/activity_pub.yaml

```diff
@@ -1,144 +1,140 @@
 ap_webfinger:
-  controller: App\Controller\ActivityPub\WebFingerController
-  path: '/.well-known/webfinger'
-  methods: [ GET ]
+    controller: App\Controller\ActivityPub\WebFingerController
+    path: '/.well-known/webfinger'
+    methods: [GET]
 
 ap_hostmeta:
-  controller: App\Controller\ActivityPub\HostMetaController
-  path: '/.well-known/host-meta'
-  methods: [ GET ]
+    controller: App\Controller\ActivityPub\HostMetaController
+    path: '/.well-known/host-meta'
+    methods: [GET]
 
 ap_node_info:
-  controller: App\Controller\ActivityPub\NodeInfoController::nodeInfo
-  path: '/.well-known/nodeinfo'
-  methods: [ GET ]
-
-ap_node_info_v20:
-  controller: App\Controller\ActivityPub\NodeInfoController::nodeInfoV20
-  path: '/nodeinfo/2.0.{_format}'
-  methods: [ GET ]
-  format: json
-
-ap_node_info_v21:
-  controller: App\Controller\ActivityPub\NodeInfoController::nodeInfoV21
-  path: '/nodeinfo/2.1.{_format}'
-  methods: [ GET ]
-  format: json
+    controller: App\Controller\ActivityPub\NodeInfoController::nodeInfo
+    path: '/.well-known/nodeinfo'
+    methods: [GET]
+
+ap_node_info_v2:
+    controller: App\Controller\ActivityPub\NodeInfoController::nodeInfoV2
+    path: '/nodeinfo/{version}.{_format}'
```

Comment:
Noe thsi line.. We combine the two nodeinfo paths to a single config, allowing to have 2.0 or 2.1 as `{version}`

Reply:
`.well-known/nodeinfo`

seems to generate a 500 for me:

```
Unable to generate a URL for the named route "ap_node_info_v21" as such route does not exist.
```

Reply:
WOH!? this is not good. Thanks for spotting.

Reply:
I forgot about that. Fixed ;)